css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      padding: 1%;
+    }
+
+    .card-container {
+      flex-basis: 16%;
+      padding: 0.5% 1%;
```

Comment:
while percentages can be used for padding and margin, it is best to use hard coded values such as px and rem as it will cause less problems with element positions in more complex layouts